packages/glsp-client/src/browser/crossmodel-diagram-module.ts

```diff
@@ -3,19 +3,24 @@
  ********************************************************************************/
 import {
    configureDefaultModelElements,
+   configureModelElement,
    ConsoleLogger,
    createDiagramContainer,
    LogLevel,
    overrideViewerOptions,
    TYPES
 } from '@eclipse-glsp/client';
 import { Container, ContainerModule } from '@theia/core/shared/inversify';
+import { EntityNodeView } from '../views';
+import { EntityNode } from '../model';
 
 const crossModelDiagramModule = new ContainerModule((bind, unbind, isBound, rebind) => {
    rebind(TYPES.ILogger).to(ConsoleLogger).inSingletonScope();
    rebind(TYPES.LogLevel).toConstantValue(LogLevel.warn);
    const context = { bind, unbind, isBound, rebind };
    configureDefaultModelElements(context);
+
+   configureModelElement(context, 'node:entity', EntityNode, EntityNodeView);
```

Comment:
Can you add some comment here what this does?

Reply:
added in the code